lib/raop_ntp.c

```diff
@@ -21,7 +21,7 @@
 #include <string.h>
 #include <stdbool.h>
 
-#include "raop_ntp.h"
```

Comment:
Why did you remote that include?

Reply:
i reversed this, and added the new declartations in a new include file raop_ports.h, include here and in raop.h